edxsearch/urls.py

```diff
@@ -8,4 +9,7 @@
 
 # urlpatterns is the standard name to use here
 # pylint: disable=invalid-name
-urlpatterns = patterns('', url(r'^search/', include(search.urls)),)
+if django.VERSION < (1, 9):
+    urlpatterns = [url(r'^search/', include(search.urls))]
```

Comment:
Django 1.9+ still supports the use of the `include()` function. It has simply been deprecated: https://docs.djangoproject.com/en/1.11/ref/urls/#include. There is no reason to fork here unless we decide, in the future, we must continue supporting Django < 1.11.